tests/Functional/App/cases/Basic/config.xml

```diff
@@ -6,8 +6,7 @@
     xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
     <content-api:content-api>
-        <content-api:service prefix="service-one" items="Libero\ContentApiBundle\Adapter\InMemoryItems"/>
-        <content-api:service prefix="service-two" items="Libero\ContentApiBundle\Adapter\NullItems"/>
+        <content-api:service prefix="service-one" include-prefix="false" items="Libero\ContentApiBundle\Adapter\InMemoryItems"/>
```

Comment:
I don't remember why we use XML for the examples here since it needs to be translated to YAML in actual services?

Reply:
Symfony allows various formats. XML is best in the tests as there's an XSD and a little extra setup for it. It would be useful to test the various formats though.